tests/src/test/java/io/snyk/languageserver/LsRuntimeEnvironmentTest.java

```diff
@@ -145,4 +145,14 @@ void testEnableTelemetryIsAddedToEnvironmentDisabled() throws StorageException {
     // This is a bit confusing - CLI takes DISABLE as env variable, but we ask for ENABLE, so it's reverted
     assertEquals("1", env.get(Preferences.ENABLE_TELEMETRY));
   }
+  @Test
+  void testAddPath() throws StorageException {
+	String expected = "C;/myPath/:";
+    HashMap<String, String> env = new HashMap<>();
+    when(preferenceMock.getPref(Preferences.PATH_KEY)).thenReturn(expected);
+    
+    environment.addPath(env);
+
+    assert(env.get("PATH").startsWith(expected));
```

Comment:
Well, wouldn't the path just be transmitted to LS, and not added to the env?

Reply:
While addPath() still lives at LsRuntimeEnviornment, it is now only used for CLI invocations. This is why it still needs to be fixed.